### PR TITLE
Add links to user guide and pre-built exp matrix

### PR DESCRIPTION
**Pull request by xiaolongchen0627**

### CODE REVIEW

File: src/core/README.md [hunk 1]
```diff
@@ -12,6 +12,7 @@
   * [MEME Suite] =4.9.0
   * [twoBitToFa]\*
   * [variants2matrix] (See below.)
+  * [pre-built expression matrix for cancers]
```
Comment (zaeleus) [inline src/core/README.md]: This is not a hard dependency. Precalculated expression matrices are discussed under the [References](https://github.com/stjude/cis-x/tree/fc8639c5bf1ab127583c18714f3150a6966bd945/src/core#references) section of the README.

**Review by zaeleus — CHANGES_REQUESTED** [review]
Delete `.DS_Store`.

File: src/core/README.md (hunk 1 above)
Comment (xiaolongchen0627) [reply src/core/README.md]: Oh, that is my bad. we can leave it as it was.

Comment (xiaolongchen0627) [reply src/core/README.md]: should I just change the README.md back and re-push it ?

Comment (zaeleus) [reply src/core/README.md]: Yes, you can edit and repush anything in your `add-a-user-guide-link` branch.

Comment (xiaolongchen0627) [reply src/core/README.md]: I change it back, should I pull another request?

Comment (zaeleus) [reply src/core/README.md]: No, changes are updated in this PR: https://github.com/stjude/cis-x/pull/2/commits/5e42db2e3b334fba3d92c9fad59bc95977c3b988

**Review by zaeleus — APPROVED** [review]

**Pull request merged**